extensions/view/README.md

```diff
@@ -0,0 +1,59 @@
+# View Geometry Extension Specification (`view`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the View Geometry Extension to a STAC Item. View Geometry adds metadata related to angles of sensors and other radiance angles that affect the view of resulting data. It will often be combined with other extensions that describe the actual data, such as the `eo`, `sat` or `sar` extensions. It is not necessary, but recommended to place common fields in [STAC Collections](../../collection-spec/collection-spec.md) using the [Commons extension](../commons/).
```

Comment:
Very minor point, but on 'It is not necessary, but recommended to place common fields in [STAC Collections]' - won't most of these be changing if they are used? The sat extension orbit stuff I think is more likely to be static. But even planet has some variation in the view angles (not sure about landsat + sentinel, but I think they can?), and the sun angles will change with the time of year. So we can maybe get rid of this line.

Reply:
Yes, true. I think we should encourage commons only for EO/SAR due to the bands fields.

Reply:
I think I just lifted this directly from the `sat` extension README.md

I can delete it.

Reply:
but maybe @matthewhanson has input on this line?

Reply:
nevermind :D